tests/test_solvermode/test_species_model.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core.examples import download_file
+
+
+@pytest.fixture(scope="module")
+def solver_session():
+    cas_path = download_file(
+        "mixing_elbow.cas.h5",
+        "pyfluent/mixing_elbow",
+    )
+    solver_session = pyfluent.launch_fluent(mode="solver")
+    solver_session.file.read_case(file_name=cas_path)
+    yield solver_session
+    solver_session.exit()
+
+
+@pytest.mark.quick
+@pytest.mark.setup
```

Comment:
We are not using these markers currently, so IMO they are just unnecessary copy-pasted repetition. I would remove them:
```suggestion
```

Reply:
What do you think, should we also remove them from `pyproject.toml`?  That's where they are defined:

https://github.com/ansys/pyfluent/blob/53aa0270d1ab0d9bdc91c0b6d749297357656c8a/pyproject.toml#L16

Reply:
That would make sense to me, the only markers we are actually using in the GitHub CIs are: `nightly`, `fluent-version`, `solvermode`, `codegen_required` and `standalone`.

In my understanding, we could remove all of these as they aren't being used, and aren't providing any helpful info that the test name or module name don't already provide:
https://github.com/ansys/pyfluent/blob/53aa0270d1ab0d9bdc91c0b6d749297357656c8a/pyproject.toml#L17-L22

Reply:
Discussion in https://github.com/ansys/pyfluent/discussions/2287

Reply:
OK, so lets leave `setup` but remove everything else that are not used currently.